backend/backend/models/tables.py

```diff
@@ -107,8 +108,14 @@ class User(UserBase, table=True):
 
 
 ### Message
+class MessageScope(str, enum.Enum):
```

Comment:
I dont see the scope in the doc https://docs.google.com/document/d/1oKfW3mX5OUcN1tsFA38AgfsQKtoPitsY76dkA4pV0Ig/edit?tab=t.0#heading=h.6a0epnyz1zl1

Could you describe what is the difference between private and public message exactly?